Source/WebCore/rendering/style/RenderStyle.h

```diff
@@ -2239,13 +2240,13 @@ class RenderStyle final : public CanMakeCheckedPtr<RenderStyle> {
     bool scrollAnchoringSuppressionStyleDidChange(const RenderStyle*) const;
     bool outOfFlowPositionStyleDidChange(const RenderStyle*) const;
 
-    static Vector<AtomString> initialAnchorNames();
-    inline const Vector<AtomString>& anchorNames() const;
-    inline void setAnchorNames(const Vector<AtomString>&);
+    static Vector<Style::AnchorName> initialAnchorNames();
```

Comment:
Nit: `Vector<Style::ScopedName>` isn't the most effective structure, since the scopeOrdinal that's being provided for each item is the same in the whole vector. View Transition classes has the same issue, so we should probably fix it there too.

Maybe we should have: 
```
struct Style::ScopedNameVector {
    Vector<AtomString> names;
    ScopeOrdinal ordinal;
}
```